README.md

```diff
@@ -1,13 +1,13 @@
 # PowerDNS Community Cookbook
 
-Provides resources for installing and configuring both PowerDNS authoritative and recursor. It uses the official PowerDNS repositories for packages and installs the appropiate configuration for your platform's init system.
+Provides resources for installing and configuring both PowerDNS authoritative and recursor. It uses the official PowerDNS repositories for packages and installs the appropriate configuration for your platform's init system.
 
 ## Build Status
 
 [![Build Status](https://travis-ci.org/dnsimple/chef-pdns.svg?branch=master)](https://travis-ci.org/dnsimple/chef-pdns)
 
 ## Upgrade Notes for 7.x series
```

Comment:
It sounds like this should be a Major since we are dropping support for things, can we clean up this header solely to focus on the breaking changes in 8.x